CONTRIBUTING.md

```diff
@@ -136,6 +137,47 @@ cases to our [automated test suite](ci/README.md) to guarantee enough
 coverage. A PR that makes significant code changes without contributing new test
 cases will be flagged by reviewers and will not be accepted.
 
+### Getting your PR verified by CI
+
+It is a requirement to get your PR verified with CI checks before it gets merged. Also, it helps to find possible bugs
+before the review work starts. Once you create a PR, or you push new commits, CI checks at the bottom of a PR page will
+be refreshed. Checks include Github Action ones and Jenkins ones. Github Action ones will be triggered automatically
+when PR is pushed and Jenkins ones need triggering with comments.
+
+The trigger phrases for one check:
```

Comment:
I don't remember if anyone can use these triggers or if you need to have write access to the repo / be listed in a Jenkins config file somewhere. Not sure what we agreed to in the end when we discussed this a while back.

Reply:
Anyone can trigger Jenkins jobs.